.github/workflows/pypi_packaging.yml

```diff
@@ -80,9 +80,6 @@ jobs:
     - name: Setup Env
       run: python3 -m pip install --upgrade pip build
 
-    - name: Validate setup.py
```

Comment:
There is no integrated solution for validating the pyproject.toml file. There are some third-party application but they are unstable. 
I've noticed though that tox run excutting its test automatically does a basic level check of the toml file using not a cli command but via package called "tomli". This has been added to the dev-requirments.txt file